app/views/community/index.html.erb

```diff
@@ -79,6 +79,12 @@
     If you need specific help about one of the for-profit Git hosting sites, you might try their own IRC channels (such as <span class="highlight fixed">#github</span> or <span class="highlight fixed">#gitlab</span>) on the same IRC server.
   </p>
 
+  <h2> News Letter </h2>
```

Comment:
I'm not 100% sure of this, but I've always seen written Newsletter and not News Letter.

https://www.quora.com/Which-is-the-correct-spelling-newsletter-or-news-letter-Why

Reply:
Thanks I changed it to "Newsletter", squashed everything and force pushed.